docs/core/compatibility/cryptography/10.0/rfc2898derivebytes-constructors.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "Breaking change: Rfc2898DeriveBytes constructors are obsolete"
+description: Learn about the .NET 10 breaking change in core .NET libraries where Rfc2898DeriveBytes constructors are obsolete.
+ms.date: 10/10/2023
+---
+# Rfc2898DeriveBytes constructors are obsolete
+
+Starting in .NET 10, all of the constructors on `Rfc2898DeriveBytes` are obsolete.
+
+## Previous behavior
+
+The `Rfc2898DeriveBytes` had constructors that were not obsolete, or obsolete under a different diagnostic ID.
+
+## New behavior
+
+The `Rfc2898DeriveBytes` constructors are obsolete with SYSLIB0060 diagnostic ID and message:
+
+> The constructors on Rfc2898DeriveBytes are obsolete. Use the static Pbkdf2 method instead.
+
+## Version introduced
+
+.NET 10 Preview 1
+
+## Type of breaking change
+
+This change is a [source incompatible](../../categories.md#source-incompatible) change.
```

Comment:
I know you removed request for review, but this wording needs to be changed to "This change can affect source compatibility". That's because it's only source incompatible if it affects your app specifically. I guess it should be changed in David's tool, ideally. Same for binary incompatible.